pkg/models/components/components.go

```diff
@@ -73,38 +73,30 @@ func GetComponentStatus(name string) (interface{}, error) {
 	}
 	for _, v := range pods {
 		component.TotalBackends++
-		component.HealthyBackends++
-		for _, c := range v.Status.ContainerStatuses {
-			if !c.Ready {
-				component.HealthyBackends--
-				break
-			}
+		if v.Status.Phase == corev1.PodRunning {
+			component.HealthyBackends++
 		}
```

Comment:
A pod is considered as healthy only if all containers inside are Ready, pod.Status.Phase is `Running` no matter what status containers are.